charts/templates/secret.yaml

```diff
@@ -6,6 +12,11 @@ metadata:
   labels:
     {{- include "marklogic.labels" . | nindent 4 }}
 type: kubernetes.io/basic-auth
+data:
+  {{- if .Values.auth.adminPassword }}
+    password: {{ .Values.auth.adminPassword }}
```

Comment:
Can we simply try to use this:
adminPassword: {{ (.Values.auth.adminPassword | default (randAlphaNum 10)) | b64enc | quote }}